Jarvis/tests/test_weatherIn.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+from Jarvis import Jarvis
+from mock import patch
+from packages import weather_pinpoint, weatherIn
+
+
+class weatherInTest(unittest.TestCase):
+
+    def setUp(self):
+        self.jarvis = Jarvis()
```

Comment:
I could use some help as I am failing to understand why calling `Jarvis()` here is an issue according to **Codacy** while I am copying this same syntax from the `test_parser` file.
Also, the tests do run correctly without breaking or throwing any errors or warnings.